pypdf/_writer.py

```diff
@@ -182,13 +181,7 @@ def __init__(
 
         # info object
         info = DictionaryObject()
-        info.update(
-            {
-                NameObject("/Producer"): create_string_object(
-                    codecs.BOM_UTF16_BE + "pypdf".encode("utf-16be")
-                )
-            }
-        )
+        info.update({NameObject("/Producer"): create_string_object("pypdf")})
```

Comment:
Could you please elaborate this change?

Reply:
Due to the change the original code passing the utf16 bom was inducing the bom to be reported in the pdf file. Although valid this was not nice and also affecting at least one test. With this mod the test/output is unchanged